data/mods/CRT_EXPANSION/items/crt_tools.json

```
@@ -86,21 +92,16 @@
     "description": "C.R.I.T R&D's masterpiece weapon.  Alien runes adorn the ridiculously oversized carbon steel blade and a hum of energy thrums from within.  Merely brushing your fingers over the weapon brings a feeling of invincibility.  It looks more like a raw heap of iron than a sword.  The thing is… can you wield it?",
     "color": "dark_gray",
     "material": [ "hardsteel" ],
-    "artifact_data": {
-      "charge_type": "ARTC_NULL",
-      "effects_wielded": [
-        "AEP_ALL_UP",
-        "AEP_STR_UP",
-        "AEP_STR_UP",
-        "AEP_CARRY_MORE",
-        "AEP_SPEED_UP",
-        "AEP_SPEED_UP",
-        "AEP_SPEED_UP",
-        "AEP_SPEED_UP",
-        "AEP_SPEED_UP",
-        "AEP_SPEED_UP",
-        "AEP_RESIST_ELECTRICITY",
-        "AEP_SAP_LIFE"
+    "//": "Legacy artifact data was set for all stats +2 Str +6, carry_more, speed up x6, resist electricity, sap hp",
+    "relic_data": {
+      "passive_effects": [
+        { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "STRENGTH", "add": 6 } ] },
+        { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "SPEED", "add": 60 } ] },
```

```suggestion
        { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "SPEED", "add": 120 } ] },
```
the legacy AEP_SPEED_UP is +20